.github/workflows/cla.yml

```diff
@@ -0,0 +1,74 @@
+name: CLA verification
+on:
+  pull_request_target:
+    types: [opened, synchronize]
+
+jobs:
+  cla-verify:
+    runs-on: ubuntu-latest
+    steps:
+      - id: 'query-worksheet'
+        uses: jroehl/gsheet.action@release
+        with:
+          spreadsheetId: "1TXZIZ_QLGagycLKt4b54FapKHvLmuIj2KHFHK_yOZDk"
```

Comment:
Probably be better as a repo secret?

Reply:
Done! 

Should we delete this comment to avoid the `spreadsheetId` visible in future?